src/main/kotlin/no/nav/klage/oppgave/api/controller/KlagebehandlingVedtakController.kt

```diff
@@ -135,9 +135,9 @@ class KlagebehandlingVedtakController(
         )
         return VedtakEditedView(
             vedtakService.setHjemler(
-                klagebehandlingId,
-                input.hjemler?.map { Hjemmel.of(it) }?.toSet() ?: emptySet(),
-                innloggetSaksbehandlerRepository.getInnloggetIdent()
+                klagebehandlingId = klagebehandlingId,
+                hjemler = input.hjemler?.map { Registreringshjemmel.of(it) }?.toSet() ?: emptySet(),
```

Comment:
Skulle vi hatt på noe feilhåndtering her, for å ta hånd om eventuelle Hjemmel-IDer som ikke finnes i Registreringshjemmel? De kunne kanskje bare vært stille droppet?

Reply:
(På sikt er det meningsløst, det er bare ment som et forslag for å håndtere en kort overgangsfase hvis frontend ikke har tilpasset seg til å bruke Registreringshjemmel ennå..)

Reply:
I prod vil nok aldri noe velge en hjemmel vi ikke har, og i dev så kan det feile hvis de gjør det?